releng/com.b2international.snowowl.server.update/snow-owl-oss.product

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <?pde version="3.5"?>
 
-<product name="Snow Owl Server" uid="com.b2international.snowowl.server" id="com.b2international.snowowl.server.product.product" application="com.b2international.snowowl.server.product.app" version="9.0.1.qualifier" useFeatures="true" includeLaunchers="true">
+<product name="Snow Owl Server" uid="com.b2international.snowowl.server" id="com.b2international.snowowl.server.product.product" application="com.b2international.snowowl.server.product.app" version="9.0.1.qualifier" type="features" includeLaunchers="true" autoIncludeRequirements="true">
```

Comment:
Tycho does not seem to support `autoIncludeRequirements` yet (at least the issue is still open): https://github.com/eclipse-tycho/tycho/issues/1056

Do we need this option, is it helping with something within the IDE?